src/HelperTrait.php

```
@@ -239,32 +239,115 @@ protected function helperExpandEntityFieldsFixtures(string $entity_type, EntityS
+
+        if (!is_file($fixture_path . $basename)) {
+          continue;
+        }
+
+        if (is_array($record)) {
+          if (array_key_exists('target_id', $record)) {
+            $records[$index]['target_id'] = $fixture_path . $basename;
+          }
+          else {
+            $records[$index][0] = $fixture_path . $basename;
+          }
```

PHPMD (2.15.0)

[error] The method helperExpandEntityFieldsFixtures uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)